psydac/api/tests/test_2d_multipatch_mapping_poisson.py

```diff
@@ -332,10 +316,10 @@ def test_poisson_2d_2_patches_dirichlet_parallel_0():
 
     D1     = mapping_1(A)
     D2     = mapping_2(B)
-
-    domain = D1.join(D2, name = 'domain',
-                bnd_minus = D1.get_boundary(axis=1, ext=1),
-                bnd_plus  = D2.get_boundary(axis=1, ext=-1))
+                
```

Comment:
Trailing white spaces.

Reply:
fixed in [d7a5053](https://github.com/pyccel/psydac/pull/309/commits/d7a5053aafcd8a4ddd8ed030763825cf93691868)